app/tailwind.config.js

```diff
@@ -51,19 +50,196 @@ module.exports = {
                     DEFAULT: 'hsl(var(--card))',
                     foreground: 'hsl(var(--card-foreground))',
                 },
-                surface: {
-                    DEFAULT: '#1a1a1a',
+
+                /*--- TOKENS –--*/
+                bg: {
+                    active: 'var(--bg-tertiary)',
                     brand: {
-                        DEFAULT: 'var(--surface-surface-brand)',
-                        secondary: 'var(--surface-surface-brand-secondary)',
+                        DEFAULT: 'var(--color-red)',
```

Comment:
We're moving away from red yes?

Reply:
I swapped to teal for now – we'll see how it goes. Would like to find a better brand color but we're rollin' with it